docs/production-setup.md

```diff
@@ -156,6 +150,25 @@ When new versions of ELMO are released, you will want to upgrade. To do so, ssh
 
 Then load the site in your browser. You should see the new version number in the page footer.
 
+### Converting an Existing Instance from MySQL to PostgreSQL
+
+1. Install PostgreSQL (see above).
+1. First upgrade to version v5.99 (`git checkout v5.99`), which is the last MySQL compatible version.
```

Comment:
We'll need to use the right version number here when we have one. Maybe we can use `release-5.X` and keep that up to date with the latest version + patch?